builder/handlebars/index.hbs

```diff
@@ -111,6 +111,9 @@
       {{#if example}}
         <div class="kss-modifier__wrapper">
           <div class="kss-modifier__heading kss-style">
+              <a href="#kssref-{{referenceURI}}-fullscreen"
+                 onclick="var el = document.getElementsByClassName('kss-{{referenceURI}}-fullscreen')[0]; screenfull.toggle(el); return false;"
+                      >Focus</a>
```

Comment:
ah, yes. I missed the `[0]`, and, yes, showing only the first of multiple examples is still wrong.